### PR TITLE
feat: Introduce g/global flag for login command

### PR DESCRIPTION
**Pull request by kamilogorek**
Closes: https://github.com/getsentry/sentry-cli/issues/120

### CODE REVIEW

File: src/commands/login.rs [hunk 1]
```diff
@@ -58,9 +63,17 @@ pub fn execute<'a>(_matches: &ArgMatches<'a>) -> Result<(), Error> {
         }
     }
 
-    update_config(&config, &token)?;
-    println!();
-    println!("Stored token in {}", config.get_filename().display());
+    let mut config_to_update = local_config;
+    if matches.is_present("global") {
+        let (global_filename, global_config) = load_global_config_file()?;
+        config_to_update = Config::from_file(global_filename, global_config)?.bind_to_process();
```
Comment (kamilogorek) [inline src/commands/login.rs]: Is there a better way to get an `Arc` reference here?

File: src/commands/login.rs [hunk 2]
```diff
@@ -58,9 +63,17 @@ pub fn execute<'a>(_matches: &ArgMatches<'a>) -> Result<(), Error> {
...
+    }
+
+    update_config(&config_to_update, &token)?;
+    println!(
+        "\nStored token in {}",
```
Comment (jan-auer) [inline src/commands/login.rs]: _nit_: Rather than leading with a newline, could you add a `println!();` above?

File: src/commands/login.rs (hunk 1 above)
Comment (jan-auer) [reply src/commands/login.rs]: Which part exactly? You could probably:

- Add a function `Config::global()?` which runs the above two lines 
- Remove bind_to_process, which you probably don't need?

That way you leave functions like `load_global_config_file` internal, and you just expose the `Config` object.

File: src/commands/login.rs (hunk 2 above)
Comment (kamilogorek) [reply src/commands/login.rs]: It was like this before and I changed it because it looked unnecessary. Will change it back.

**Pull request merged**